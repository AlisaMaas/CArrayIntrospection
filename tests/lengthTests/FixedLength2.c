#include <stdlib.h>

int bar() {
   return 4*rand()%30;
}

/**
* Check that randomness is properly treated as random.
**/
int foo(int* array, int x){
	int index = 10;
	int sum = 0;
	for (int i = 0; i < index; i++) {
		sum += array[i];
	}
	array[bar()] = 5;
	return sum;
	
}

