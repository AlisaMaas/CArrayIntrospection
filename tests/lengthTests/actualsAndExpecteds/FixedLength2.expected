Analyzing bar
Analyzing rand
Analyzing foo
	Argument array has unknown max index.
	Argument x has unknown max index.
